pkg/generate/app/environmentfile.go

```diff
@@ -0,0 +1,24 @@
+package app
+
+import (
+	"bufio"
+	"os"
+)
+
+// GetEnvironmentVars reads a whole file into memory and returns string
+// array. This could be a problem if we are reading a very large file. I am
+// not putting any validation for file, if it contains a key=value pair as
+// that should be handled in environment variables that we are passing.
+func GetEnvironmentVars(path string) ([]string, error) {
```

Comment:
Makes sense, in fact, in one of my earlier commits, I mentioned this in the comments. I will update it.